dali/pipeline/data/tensor.h

```diff
@@ -284,12 +297,13 @@ class Tensor : public Buffer<Backend> {
    * in use by the Tensor.
    */
   inline void ShareData(void *ptr, size_t bytes, bool pinned = false,
-                        DALIDataType type = DALI_NO_TYPE) {
-    ShareData(ptr, bytes, pinned, { 0 }, type);
+                        DALIDataType type = DALI_NO_TYPE,
+                        AccessOrder order = {}) {
+    ShareData(ptr, bytes, pinned, { 0 }, type, order);
   }
 
-  inline void Reset() {
-    reset();  // free the underlying buffer
+  inline void Reset(AccessOrder stream = {}) {
+    reset(stream);  // free the underlying buffer
```

Comment:
Why it doesn't use `order_`?

Reply:
I don't think it needs it at all at this point.